lib/tasks_private/lib/aws_products_data_collector.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+# https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/using-ppslong.html
+
+class AwsProductsDataCollector
+  OFFERS_HOSTNAME = 'https://pricing.us-east-1.amazonaws.com'
+
+  cattr_accessor :cache, :instance_writer => false do
+    cache_dir = Rails.root.join('tmp', 'aws_cache', 'products_data')
```

Comment:
Could you use a `Tmpdir` that you wouldn't have to have a separate cleanup rake task?